core/tests/protractor/learnerDashboard.js

```diff
@@ -56,16 +60,9 @@ describe('Learner dashboard functionality', function() {
     browser.waitForAngular();
     element(by.css('.protractor-test-create-activity')).click();
     // Create new collection.
-    element(by.css('.protractor-test-create-collection')).click();
+    learnerDashboardPage.createExploration();
```

Comment:
This action is not something that's being done on the learner dashboard page, so it should not be part of the learner dashboard page object.

Reply:
ok, I have removed it